From 4838111e233886e3c4ada5a570d3fbdfd6748cf5 Mon Sep 17 00:00:00 2001
From: Taylor Southwick <twsouthwick@outlook.com>
Date: Sun, 27 Oct 2019 16:19:45 -0700
Subject: [PATCH 4/4] Remove libxslt and libxml2 dir

---
 win/makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/win/makefile b/win/makefile
index 64003d1a..2cd6eeaa 100644
--- a/win/makefile
+++ b/win/makefile
@@ -33,11 +33,9 @@ ICU_INCLUDE = $(ICU_BASE)\include
 
 # libxslt
 HAVE_LIBXSLT=1
-LIBXSLT_DIR=c:\libxslt-1.1.28.win$(BARCH)
 
 # libxml2
 HAVE_LIBXML2=1
-LIBXML2_DIR=c:\libxml2-2.9.2.win$(BARCH)
 
 # get WIN32 binaries libxml2 & iconv & zlib from here:
 #  http://www.zlatkovic.com/libxml.en.html
-- 
2.19.1.windows.1

